### PR TITLE
fix: token refresh for outreach + autopause for other providers

### PR DESCRIPTION
**Pull request by asdfryan**

### CODE REVIEW

File: packages/core/remotes/impl/outreach/index.ts [hunk 1]
```diff
@@ -1413,7 +1413,7 @@ class OutreachClient extends AbstractEngagementRemoteClient {
             client_id: this.#credentials.clientId,
             client_secret: this.#credentials.clientSecret,
             grant_type: 'refresh_token',
-            refresh_token: 'asdf',
+            refresh_token: this.#credentials.refreshToken,
```
Comment (asdfryan) [inline packages/core/remotes/impl/outreach/index.ts:1416]: This is the fix. I forgot to undo the change after testing.

**Review by tomkit — APPROVED** [review]
